components/ee/db-sync/src/export.ts

```diff
@@ -83,7 +82,7 @@ export class TableUpdate {
                     .pipe(new Transform({
                         objectMode: true,
                         transform: (data: any, encoding, cb) => {
-                            if(data[this.table.timeColumn] > now()) {
+                            if(data[this.table.timeColumn] > Date.now()) {
```

Comment:
Is `timeColumn` not a mysql date string (e.g. `1970-01-01 00:00:01.001`)?

Reply:
interesting question. I didn't pay much attention to the context, just made sure that it's same value kind:

```
> moment.now()
1622818434314
```

For example L.69 also assumes, that dates column type is a unix timestamp. 
Maybe that's, what `query`'s result returns?